library-app/src/components/Sidebar/Sidebar.tsx

```diff
@@ -5,8 +5,12 @@ import account from '../../assets/icons/account.png' // loader comp za slike
 import showMore from '../../assets/icons/showMore.png'
 import { useNavigate } from 'react-router-dom'
 import { MainLayoutProps } from '../Layout/MainLayout'
+import Modal from '../Layout/Modal'
+import ManageBookForm from '../Books/BooksList/ManageBookForm'
 
 const Sidebar = ({ isLoggedIn }: MainLayoutProps) => {
+  const [isOpen, setIsOpen] = useState(false)
```

Comment:
Maybe you can rename this, because from this name it's not clear enough what it should open

Reply:
renamed in isModalOpened